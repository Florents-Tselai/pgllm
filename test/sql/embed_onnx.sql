select array_length(llm_embed('hello world', 'onnx-bge-micro'), 1);
select array_length(llm_embed('hello world', 'onnx-gte-tiny'), 1);
select array_length(llm_embed('hello world', 'onnx-minilm-l6'), 1);
select array_length(llm_embed('hello world', 'onnx-minilm-l12'), 1);
