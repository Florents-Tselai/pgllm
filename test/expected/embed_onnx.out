select array_length(llm_embed('hello world', 'onnx-bge-micro'), 1);
 array_length 
--------------
          384
(1 row)

select array_length(llm_embed('hello world', 'onnx-gte-tiny'), 1);
 array_length 
--------------
          384
(1 row)

select array_length(llm_embed('hello world', 'onnx-minilm-l6'), 1);
 array_length 
--------------
          384
(1 row)

select array_length(llm_embed('hello world', 'onnx-minilm-l12'), 1);
 array_length 
--------------
          384
(1 row)

